boards/ct/ctcc_nrf52840/board.yml

```diff
@@ -0,0 +1,10 @@
+board:
+  name: ctcc_nrf52840
+  vendor: ct
+  revision:
+    format: letter
+    default: "A"
+    revisions:
+      - name: "A"
```

Comment:
I don't think you need this if there's only one revision out

Reply:
removed